pandas/tests/test_expressions.py

```diff
@@ -431,3 +430,25 @@ def test_bool_ops_column_name_dtype(self, test_input, expected):
         # GH 22383 - .ne fails if columns containing column name 'dtype'
         result = test_input.loc[:, ["a", "dtype"]].ne(test_input.loc[:, ["a", "dtype"]])
         assert_frame_equal(result, expected)
+
+    # Can't check floordiv here because it currently doesn't work #GH27636
```

Comment:
Can we add this as an `xfail` value in the parameterization?

Reply:
Added, had to use imperatively as it only fails with axis=1 and floordiv.